docs/utilities/jasix.mdx

```diff
@@ -0,0 +1,299 @@
+Jasix                                               (c) 2017 Illumina, Inc.
+Stromberg, Roy, Lajugie, Jiang, Li, and Kang                          2.0.0
+---------------------------------------------------------------------------
+
+Ref Sequence chrM indexed in 00:00:00.2
+Ref Sequence chr1 indexed in 00:00:05.8
+Ref Sequence chr2 indexed in 00:00:06.0
+.
+.
+.
+Peak memory usage: 28.5 MB
+Time: 00:01:14.8
+```
+## Querying the index
+The Jasix query format is chr:start-end. If not provided, it assumes end=start. If only chr is provided, all entries for that chromosome will be provided.
+
+```scss
+dotnet Jasix.dll -i  input.json.gz chrM:5000-7000
+```
+
+```JSON
+{
+   "positions":[
+      {
+         "chromosome":"chrM",
+         "refAllele":"C",
+         "position":5581,
+         "quality":3070.00,
+         "filters":[
+            "LowGQXHomSNP"
+         ],
+         "altAlleles":[
+            "T"
+         ],
+         "samples":[
+            {
+               "variantFreq":1,
+               "totalDepth":1625,
+               "genotypeQuality":1,
+               "alleleDepths":[
+                  0,
+                  1625
+               ],
+               "genotype":"1/1"
+            }
+         ],
+         "variants":[
+            {
+               "altAllele":"T",
+               "refAllele":"C",
+               "begin":5581,
+               "chromosome":"chrM",
+               "end":5581,
+               "variantType":"SNV",
+               "vid":"MT:5581:T"
+            }
+         ]
+      },
+      {
+         "chromosome":"chrM",
+         "refAllele":"A",
+         "position":6267,
+         "quality":1637.00,
+         "filters":[
+            "LowGQXHetSNP"
+         ],
+         "altAlleles":[
+            "G"
+         ],
+         "samples":[
+            {
+               "variantFreq":0.6873,
+               "totalDepth":323,
+               "genotypeQuality":1,
+               "alleleDepths":[
+                  101,
+                  222
+               ],
+               "genotype":"0/1"
+            }
+         ],
+         "variants":[
+            {
+               "altAllele":"G",
+               "refAllele":"A",
+               "begin":6267,
+               "chromosome":"chrM",
+               "end":6267,
+               "variantType":"SNV",
+               "vid":"MT:6267:G"
+            }
+         ]
+      }
+   ]
+}
+
+```
+The default output stream is Console. However, if an output filename is provided, Jasix outputs the results to that file in a bgzipped format. The output is always a valid JSON entry. If requested (via -t option) the header of the indexed file will be provided. Multiple queries can be submitted in the same command and the output will contain them within the same "positions" block in order of the submitted queries (Warning: if the queries are out of order, or overlapping, the output will be out or order and intersecting).
```

Comment:
`bgzipped format` is our own informal way of saying this. Maybe `bgzip compressed format` would be better.